src/main/resources/i18n/messages.properties

```diff
@@ -1,14 +1,14 @@
 app.title=Vetlog
 app.mailto=mailto:vetlog@josdem.io
 app.contact.us=Contact Us: vetlog@josdem.io
-app.description=Only in Mexico half a million dogs are euthanized every single year. Help us to decrease that number adopting instead!
+app.description=Only in Mexico are half a million dogs euthanized every single year. Help us to decrease that number by adopting instead!
```

Comment:
app.description=Only in Mexico half a million dogs are euthanized every single year. Help us to decrease that number by adopting instead!


This is correct

Reply:
yeah. you're right. thanks